src/App.js

```diff
@@ -1,25 +1,21 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+// components
+import PostList from './components/post-list';
+
+// mock data
+import posts from './data/posts.json';
+import users from './data/users.json';
+
+const App = () => {
+    return (
+        <>
+            <header className="App-header">
+                <h1>Blog Demo</h1>
+            </header>
+            <PostList posts={posts} users={users} />
+        </>
```

Comment:
This block is contained within what look like empty HTML tags (<> & </>). Could you explain why you have formatted it that way?

Reply:
The empty tags are [shorthands](https://reactjs.org/docs/fragments.html#short-syntax) for [`Fragments`](https://reactjs.org/docs/fragments.html) in React, which are used when a component returns more than one DOM child and a `div` or any other tag is not really applicable.

In this example, the `App` component is returning 2 DOM children: a regular `header` element and a custom tag component `PostList`.

When the component that returns a Fragment gets rendered, these tags are ignored and its content gets added to the DOM tree:
![image](https://user-images.githubusercontent.com/5166837/113830689-c57e6300-977e-11eb-9dfa-cfc14b5468b9.png)

Immediately after the root div (where the app is injected), the content of the main component is rendered... you can see that in the actual DOM tree - in the browser, there's no empty tag been injected, but instead just the header, and then the content of the `PostList`